lib/lndclient/LndClient.ts

```diff
@@ -245,8 +245,11 @@ class LndClient extends BaseClient {
   /**
    * Return the total balance available across all channels.
    */
-  public channelBalance = (): Promise<lndrpc.ChannelBalanceResponse> => {
-    return this.unaryCall<lndrpc.ChannelBalanceRequest, lndrpc.ChannelBalanceResponse>('channelBalance', new lndrpc.ChannelBalanceRequest());
+  public channelBalance = async (): Promise<lndrpc.ChannelBalanceResponse.AsObject> => {
```

Comment:
Why going back to use AsObject? is there a reason for it?

Reply:
I think that's the right idea but then maybe let's define this method signature on the `BaseClient` class, then override it here. I also think that rather than using the `AsObject` type we can define an internal type that mirrors it, just so that the Raiden client isn't relying on type definitions from lnd.

Reply:
Done. `BaseClient` now has the method signature.